src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //равнобедренная трапеция
+    private double height; // высота
+
+    public IsoscelesTrapezoid(String color, double firsSide, double secondSide, double height) {
+        super(color, firsSide, secondSide);
+        this.height = height;
+    }
+
+    @Override
+    public double detAreaOfFigure() {
```

Comment:
There is a typo in the method name `getFirsSide`. It should be `getFirstSide` to match the corrected variable name in the `FigureTwo` class.